src/script.js

```diff
@@ -4,34 +4,9 @@ createTheModal = () => {
   // Get the <span> element that closes the modal
   const span = document.getElementsByClassName("close")[0];
 
-  // When the user clicks on the button, open the modal
-  globalButton.onclick = () => {
-    modal.style.display = "block";
-    const ol = document.getElementById("modalInformation");
-    const li = document.createElement("li");
-    ol.appendChild(li);
-    const globalInfo = document.createTextNode('Starting March 29, 2020');
-    const globalGoal = document.createTextNode('DEMO PROJECT')
-    li.append(globalInfo);
-    li.append(globalGoal);
-  }
-
-  wozButton.onclick = () => {
-    modal.style.display = "block";
-    const wozOl = document.getElementById("wozModalInformation");
-    const wozLi = document.createElement("li");
-    wozOl.appendChild(wozLi);
-    const wozInfo = document.innerHTML('Graduated November 5, 2018 With a full stack Certificate');
-    wozLi.append(wozInfo);
-  }
-
-  mpgButton.onclick = () => {
-    modal.style.display = "block";
-  }
-
-  // When the user clicks on <span> (x), close the modal
   span.onclick = () => {
     modal.style.display = "none";
+    console.log('modal', modal);
```

Comment:
This naming is also a bit weird. The id is "modal" in the html, but you set it the var name in line 2 to "globalModal," and reference both of them in this js. Are they different?